_data/toc.yaml

```diff
@@ -1162,6 +1162,8 @@ manuals:
       title: Quick Start Guide and sign in
     - sectiontitle: Explore Docker Desktop
       section:
+        - path: /desktop/use-desktop/overview/
+          title: Overview
```

Comment:
Can we make this `/desktop/use-desktop/` (without the `/overview/` suffix?) That way there's a logical hierarchy in the URLs;

- `/desktop/use-desktop/` (overview)
     - -> `/desktop/use-desktop/container/`
     - -> `/desktop/use-desktop/images/`
     - -> `/desktop/use-desktop/volumes/`
     - etc.


With that, I can be on a "details" page, and move "one level up" in the URL path, and arrive at the overview for the "explore" section; move one level up, and arrive at the overview for the "desktop" section, etc.